steps/24_categorical_support/step.md

```diff
@@ -0,0 +1,143 @@
+# Adding categorical support to sktime
+
+Contributors: @Abhay-Lejith
+
+## Introduction
+
+### Objectives 
+Allow users to pass data with categorical features to any estimator. If not supported, return an informative error message. Otherwise it should run without errors.
+
+### Current status:
+* Non-numeric dtypes are completely blocked off in datatype checks.
+* https://github.com/sktime/sktime/pull/5886 - This pr removes these checks but has not been merged yet as most estimators do not support categorical and will break without informative error messages.
+* https://github.com/sktime/sktime/pull/6490 - pr adding feature_kind metadata
+
+
+## Description of proposed designs
+
+
+For each design we will have to discuss 8 combinations of cases:
```

Comment:
try to give more context to the reader, what is the context/setting? no.2 in particular may be unclear without